frontend/src/Initialize.tsx

```diff
@@ -216,6 +217,146 @@ const Initialize = (props: Props) => {
+                SetGameMode: value,
+              },
+            });
+            break;
+          case "num_decks":
+            send({
+              Action: {
+                SetNumDecks: value,
+              },
+            });
+            if (kittySizeSet) {
+              // reset the size again, as setting deck numn resets kitty_size to default
+              send({
+                Action: {
+                  SetKittySize: kittySize,
+                },
+              });
+            }
+            break;
+          case "kitty_size":
+            send({
+              Action: {
+                SetKittySize: value,
+              },
+            });
+            kittySizeSet = true;
+            kittySize = value;
+            break;
+          case "friend_selection_policy":
+            send({
+              Action: {
+                SetFriendSelectionPolicy: value,
+              },
+            });
+            break;
+          case "hide_landlord_points":
+            send({
+              Action: {
+                SetHideLandlordsPoints: value,
+              },
+            });
+            break;
+          case "hide_played_cards":
+            send({ Action: { SetHidePlayedCards: value } });
+            break;
+          case "advancement_policy":
+            send({
+              Action: {
+                SetAdvancementPolicy: value,
+              },
+            });
+            break;
+          case "kitty_bid_policy":
+            send({
+              Action: {
+                SetKittyBidPolicy: value,
+              },
+            });
+            break;
+          case "kitty_penalty":
+            send({
+              Action: {
+                SetKittyPenalty: value,
+              },
+            });
+            break;
+          case "throw_penalty":
+            send({
+              Action: {
+                SetThrowPenalty: value,
+              },
+            });
+            break;
+          case "trick_draw_policy":
+            send({
+              Action: {
+                SetTrickDrawPolicy: value,
+              },
+            });
+            break;
+          case "throw_evaluation_policy":
+            send({
+              Action: {
+                SetThrowEvaluationPolicy: value,
+              },
+            });
+            break;
+        }
+      }
+    }
+  };
+
+  const loadGameSettings = (evt: any) => {
+    evt.preventDefault();
+    const settings = localStorage.getItem("gameSettingsInLocalStorage");
+    if (settings !== null) {
+      let gameSettings: IPropagatedState;
+      try {
+        gameSettings = JSON.parse(settings);
+        setGameSettings(gameSettings);
+      } catch (err) {
+        localStorage.setItem(
+          "gameSettingsInLocalStorage",
+          JSON.stringify(props.state.propagated)
+        );
+      }
+    }
+  };
+
+  const resetGameSettings = (evt: any) => {
+    evt.preventDefault();
+
+    fetch("/default_settings")
```

Comment:
1. any chance we can rename `default_settings` to `default_settings.json`?
2. should we be using a relative path here? the deployed URL is in a subdirectory

Reply:
I think this still needs to be updated :)

Reply:
Sure. I was only following your convention of stats not stats.json :)